src/uct/ib/mlx5/gga/gga_mlx5.c

```diff
@@ -848,10 +848,10 @@ static uct_md_ops_t uct_mlx5_gga_md_ops = {
     .query              = uct_ib_mlx5_gga_md_query,
     .mem_alloc          = uct_ib_mlx5_devx_device_mem_alloc,
     .mem_free           = uct_ib_mlx5_devx_device_mem_free,
+    .mem_advise         = uct_ib_mem_advise,
```

Comment:
The changes in this file can be reverted.

Reply:
it's indeed a minor change to move the field location, but it's to align all struct definitions to the same order. i can move it to another PR, WDYT?

Reply:
I do not mind to change the order in this PR. However, I see that `mkey_pack` and `mem_attach` go in different sequences in different structures.

Reply:
right, fixed in gga, ib, and cuda_copy